src/components/NftPreview/NftPreview.tsx

```diff
@@ -99,13 +101,33 @@ function NftPreview({ galleryNftRef }: Props) {
     }
   }, [columns, aspectRatioType, isMobile]);
 
+  const result = getVideoOrImageUrlForNftPreview(nft);
+
+  const nftPreviewWidth = useMemo(() => {
+    // this allows SVGs to stretch to fit its container, fixing images
+    // that appeared tiny.
+    //
+    // HOWEVER, stretching an svg to 100% when column size = 1 results
+    // in the preview label appearing stretched beneath the image, since
+    // we cap the max height to 80vh when column = 1; so this is disabled
+    // in those cases for now.
+    if (columns > 1 && result?.urls?.large?.endsWith('.svg')) {
+      return '100%';
+    }
+    return 'auto';
+  }, [columns, result?.urls?.large]);
```

Comment:
this causes a regression in 1-column collections with SVGs in terms of the floating label: 
<img width="1108" alt="image" src="https://user-images.githubusercontent.com/12162433/166316210-3ed2cc06-ee49-4cfa-8141-37dbfaf8d1f0.png">

in other words,
1) IF `octal` decides to display his bird in single-column view, it would still appear very small.
2) BUT fixing this isn't straightforward and it would cause a regression in other galleries

this PR compromises by fixing SVG appearance in 1+ column layout views

Reply:
nice! good compromise. maybe medium term we can just separate out the logic for rendering single column nfts specifically since that seems to be the case that's giving us the most edge cases. that should fix svg sizes for single-column view once and for all